src/helpers/queries.js

```diff
@@ -162,11 +162,14 @@ export function getCommentsByPlan(id) {
 
 export function getCommitteeMeetings(committeeIds) {
   return `query getCommitteeMeetings {
-    meetings(where: { committee_in: ${JSON.stringify(committeeIds)} }) {
+    meetings(where: { committee_in: ${JSON.stringify(
+      committeeIds
+    )}, isHidden: "false"}) {
```

Comment:
shouldn't it be `isHidden: false`? (without " ")

Reply:
It's not working without the quotes

Reply:
I just remembered - we need to use an operator..
So it should work like this: `isHidden_ne: true` 
(which means "isHidden not equals true")

Reply:
updated in branch